bitnami/discourse/templates/_helpers.tpl

```diff
@@ -243,18 +254,35 @@ Return the PostgreSQL User
 {{- end -}}
 
 {{/*
-Return the PostgreSQL Secret
+Return true if a secret object for Postgres should be created
+*/}}
+{{- define "discourse.postgresql.createSecret" -}}
+{{- if and (not .Values.postgresql.enabled) (not .Values.externalDatabase.existingSecret) }}
+    {{- false -}}
```

Comment:
I think here is the other way around. This is `true` and the else statement should be `false`